src/components/RightSidebar/TableOfContents.tsx

```diff
@@ -125,16 +131,18 @@ const TableOfContents: FunctionComponent<Props> = ({ toc = [], labels, isMobile
 
 	return (
 		<Container>
-			<HeadingContainer>
-				<h2 className="heading" id={onThisPageID}>
-					{labels.onThisPage}
-				</h2>
-			</HeadingContainer>
-			<ul className="toc-root">
-				{toc.map((heading) => (
-					<TableOfContentsItem key={heading.slug} heading={heading} />
-				))}
-			</ul>
+			<>
```

Comment:
We can avoid this unnecessary fragment wrapper by using Preact’s `ComponentChildren` type for the `Container` component:

```js
const Container = ({ children }: { children: ComponentChildren }) => {
```

Reply:
We would have this error then:

<img width="829" alt="" src="https://user-images.githubusercontent.com/1091472/213423620-0ad518a6-2a89-45b9-abee-fc3a722b0a0d.png">

Any suggest?

Reply:
Ahh right because in one branch of the component we return `children` directly. We can probably use a Fragment inside `Container` in that case. I think that’s the best solution:

```jsx
	const Container = ({ children }: { children: ComponentChildren }) => {
		return isMobile ? (
			<details
				{...{ open }}
				onToggle={(e: JSX.TargetedEvent<HTMLDetailsElement>) => setOpen(e.currentTarget.open)}
				className="toc-mobile-container"
			>
				{children}
			</details>
		) : (
			<>{children}</>
		);
	};
```

Reply:
Done.